src/pages/Home/index.jsx

```diff
@@ -1,22 +1,28 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import Box from '@material-ui/core/Box';
 import Grid from '@material-ui/core/Grid';
 import { Logo, SmallLogo } from './style';
 import VideoCard from '../../components/VideoCard';
 import logo from '../../utils/assets/logo.gif';
 import smallLogo from '../../utils/assets/small-logo.gif';
-import videos from '../../utils/youtube-videos-mock.json';
-import filterVideos from '../../utils/filterVideos';
+import { store } from '../../state/store';
+// import videos from '../../utils/youtube-videos-mock.json';
 
 const HomePage = () => {
+  const globalState = useContext(store);
+  const videos =
+    globalState.state.videoList.items === undefined
+      ? []
+      : globalState.state.videoList.items;
```

Comment:
With this kind of pattern, something like the following should make the trick
```suggestion
globalState.state.videoList.items || [];
```